doc/lua_api.txt

```diff
@@ -4275,9 +4275,14 @@ Setting-related
 Authentication
 --------------
 
-* `minetest.string_to_privs(str)`: returns `{priv1=true,...}`
-* `minetest.privs_to_string(privs)`: returns `"priv1,priv2,..."`
-    * Convert between two privilege representations
+* `minetest.string_to_privs(str, delim)`:
+    * Converts string representation of privs into table form
+    * `delim`: String separating the privs. Defaults to `","`.
+    * Returns `{ priv1 = true, ... }`
+* `minetest.privs_to_string(privs, delim)`:
+    * Convert table form of privs into string representation
+    * `delim`: String to delimit privs. Defaults to `","`.
+    * Returns `"priv1,priv2,..."`
```

Comment:
This would be correct only if `delim` was omitted or set to `","`. Should I replace this with a description of the return value instead?

Reply:
If it's optional this is enough:
```
* `minetest.privs_to_string(privs[, delim])`
...
    * `delim`: Optional string to delimit privileges. Defaults to `","`.
```

Reply:
Do you mean the `Returns ...` line can be omitted?

Reply:
Not that, but a generic description would be more correct.
Maybe something like `Returns the privileges concatenated to a string`? The delimiter is already explained a line above, so that's self-explaining.

Reply:
Hmm, but that's already mentioned in the `Convert table form into ...` line. I think we can combine the `Returns ...` line with this line like so:

```markdown
Returns the string representation of `privs`
```

Thoughts?

Reply:
Done